docker-compose.test.yml

```
@@ -0,0 +1,32 @@
+      POSTGRES_PASSWORD: test_password
+      POSTGRES_DB: test_payloadcms
+    volumes:
+      - postgres_test_data:/var/lib/postgresql/data
+      - ./docker/postgres-init.sql:/docker-entrypoint-initdb.d/01-init.sql:ro
+    # Healthcheck : CRUCIAL pour garantir que la BDD est prête
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -U test_user -d test_payloadcms"]
+      interval: 5s
+      timeout: 5s
+      retries: 10
+      start_period: 15s # Donne du temps à Postgres pour démarrer
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)